styles.css

```diff
@@ -0,0 +1,116 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap');
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background: #191765;
+  font-family: 'Roboto', sans-serif;
+}
+
+a {
+  text-decoration: none;
+}
+
+button {
+  cursor: pointer;
+  display: inline-block;
+  border: 0;
+  font-weight: bold;
+  padding: 10px 20px;
+  background: #191765;
+  color: #fff;
+  font-size: 15px;
+}
+
+/* Geometric shapes */
+#circle {
+  width: 160px;
+  height: 160px;
+  background: #edecff;
+  border-radius: 50%;
+  position: absolute;
+  top: 97px;
+  right: 0;
+  left: -106px;
+  z-index: -1;
+  filter: blur(30px);
+}
+
+/* Header and Navbar styles */
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 50px;
+  padding: 20px 0;
+}
+
+.logo {
+  font-weight: bold;
+  padding: 10px 20px;
```

Comment:
Nice work implementing most of the changes suggested by the previous reviewer. Your portfolio looks wonderful.

- [ ] (Optional) I would like to suggest that you change the `font-weight` from bold to `500`. This way, you would totally be following the design template provided by Figma.